src/projects/pull-request/set-reviewers.html

```diff
@@ -55,7 +55,19 @@
         reviewer: {
           type: String,
           notify: true
+        },
+        reviewers: {
+          type: Array,
+          computed: '_getReviewers(project.assignees, owner)'
```

Comment:
Yes, we fetch the possible reviewers here, and there is no separate list for that only the list with possible assignees.